### PR TITLE
Contributing docs

### PR DESCRIPTION
**Pull request by yanivefraim**
Update tests section

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,8 +22,9 @@ Please **ask first** if somebody else is already working on this or the core dev
 
 That's it, you're good to go.
 
-- `yarn test:templates` - Create all `create-yoshi-app`'s templates, install, build and test each one of them.
-- `yarn test:integration` - Create a few complex projects that cover a lot of different edge-cases, build, run, and test that everything is working.
+- `npx jest --runInBand` - Run integration all tests (not recommended locally). Please see below how to run those localy.
```
Comment (amitdahan) [inline CONTRIBUTING.md]: Whoopsie
```suggestion
- `npx jest --runInBand` - Run all integration tests (not recommended locally). Please see below how to run those locally.
```

Comment (yanivefraim) [reply CONTRIBUTING.md]: thanks!
